server/app/assets/javascripts/azure_upload.ts

```diff
@@ -3,9 +3,13 @@
  */
 
 class AzureUploadController {
-  static FILEUPLOAD_FORM_ID = 'cf-block-form'
+  private static FILEUPLOAD_FORM_ID = 'cf-block-form'
+  private static AZURE_UPLOAD = '.azure-upload'
```

Comment:
suggestion (blocking): please rename to `AZURE_UPLOAD_SELECTOR`

Reply:
Done.